front/src/routes/QuestionsPage.svelte

```diff
@@ -90,14 +103,61 @@
         loading = false;
         showError();
       });
-  }
+  };
+
+  const openConfirmationModal = () => {
+    modalText = `Are you sure you want delete these questions?`;
+    modalHeadingText = 'Delete selected questions';
+    changeStatus = () => {
```

Comment:
no need for that. I did in on `ApplicantPage` because I passed in different functions, here you can just pass `deleteQuestions` to `onSubmit` function to `ConfirmationModal`